simpl_eeg/topomap_2d.py

```diff
@@ -1,25 +1,33 @@
 import matplotlib.pyplot as plt
 import matplotlib.animation as animation
 import mne
+import numpy
 import numpy as np
 from mpl_toolkits.axes_grid1.axes_divider import make_axes_locatable
 
 
 def add_timestamp(epoch, frame_number, xpos, ypos):
     """
     Adds a timestamp to a matplotlib.image.AxesImage object
-
-    Parameters:
-        epoch: mne.epochs.Epochs
-            MNE epochs object containing the timestamps.
-        frame_number: int
-            The timestamp to plot
-        xpos: float
-            The matplotlib x coordinate of the timestamp.
-        ypos: float
-            The matplotlib y coordinate of the timestamp.
+    
+    Parameters
```

Comment:
All the formatting has been changed to the new style. Would you be able to change it back? Otherwise it doesn't show up properly on jupyter book.